### PR TITLE
AppCompatActivity and Toolbar AppCompatChanges

### PR DESCRIPTION
**Pull request by kunall17**
This changes the current old Activity to AppCompatActivity and using the toolbar.
Also uses the new DayNight AppCompat theme therefore in another PR will add a feature to switch between day and night themes: 

API 22             |  API 16
:-------------------------:|:-------------------------:
 |

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/build.gradle [hunk 1]
```diff
@@ -45,9 +45,9 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:support-v13:23.3.0'
-    compile 'com.android.support:appcompat-v7:23.3.0'
-    compile "com.android.support:recyclerview-v7:23.0.1"
+    compile 'com.android.support:appcompat-v7:24.0.0'
+    compile 'com.android.support:design:24.0.0'
+    compile 'com.android.support:recyclerview-v7:24.0.0'
```
Comment (niftynei) [inline app/build.gradle]: 🎉

File: app/src/main/java/com/zulip/android/activities/LoginActivity.java [hunk 2]
```diff
@@ -53,6 +55,10 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.login);
 
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        toolbar.setTitle("Login");
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/activities/LoginActivity.java]: Human readable strings belong in `strings.xml`.

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 3]
```diff
@@ -255,15 +257,16 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        this.onPrepareOptionsMenu(menu);
-
-        this.logged_in = true;
-
         setContentView(R.layout.main);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setHomeButtonEnabled(true);
+        getSupportActionBar().setTitle("Zulip");
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Human readable strings belong in `strings.xml`.

**Comment by niftynei** [comment]
@kunall17 this looks pretty good!  The `Welcome to Zulip` text on my screen is too small (see screenshot).  Also the checkbox & textview fields have a purplish tint to them.  Can we make them go back to black?

screenshot_20160718-195029

**Comment by kunall17** [comment]
@niftynei yeah for the accent color I changed it to purple color as in the zulip web-UI logo background color! 
what should this be?

**Comment by niftynei** [comment]
@kunall17 cool, it's a nice color.  I guess my biggest problem with it is that when you check and uncheck the Use Zulip.com box, it changes back to black.

In the future it'd be nicer to have small changes like this (changing the color of the accents) in separate PRs, with the changes explicitly labeled.  Having them all together makes it harder to figure out what's intentional and what's incidental.

**Comment by kunall17** [comment]
@niftynei updated the PR :)

**Comment by niftynei** [comment]
merged 👍

**Pull request merged**